api/pom.xml

```diff
@@ -116,62 +161,78 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <!-- Adds the manifest file created by the org.apache.felix:maven-bundle-plugin -->
             <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-jar-plugin</artifactId>
-                <version>3.1.0</version>
+                <version>2.4</version>
```

Comment:
Is this one intended to be downgraded?

Reply:
No. I was aligning JSP with EL and didn't notice the downgrade. I'll go back to EL, check I can update it there and then remove this change from the PR. Thanks for catching this.